jwst/datamodels/schemas/ifucube.schema.yaml

```diff
@@ -30,6 +30,10 @@ allOf:
               title: EXTNAME of the extension containing data quality mask
               type: string
               fits_keyword: MASKEXT
+            wave_extension:
```

Comment:
Is there a reason to have a FITS keyword pointing to the FITS extension?  Setting
```
CTYPE3  = 'WAVE-TAB'           / Wavelength from lookup table
PS3_0   = 'WCS-TAB'            / WAVE-TAB extension name
PS3_1   = 'wavelength_column_name'         / Column name of lookup table
```
Should take care of where this FITS extension lives, right?  CTYPE3 tells you there's a WAVE-TAB, PS3_0 tells you where to find it in the FITS file, and PS3_1 indicates the column to find the wavelengths.

Reply:
As I understand it this is the fits standard recommended by the Spectral Fits standard paper.

Reply:
The CTYPEn, PSi_0, and PSi_1 keywords as described by @jdavies-st are what's required by the FITS WCS standard. There's no need for an additional ad hoc keyword like "WAVEEXT". The FITS standard doesn't care about a keyword of that name. The CTYPEn, PSi_0, and PSi_1 keywords should be handled by or bundled with the rest of the WCS-related keywords contained in the header of the IFU cube HDU. As such I would expect them to be handled/created by the underlying gwcs/wcslib functionality.

Reply:
All of the keywords FLUXEXT, ERREXT, ERRTYPE, and MASKEXT were included here because they conform to conventions used in the data products for other IFU instruments (e.g. groundbased IFU's). There's no requirement for them and the only product that includes them is the IFU cubes (we don't put them in any other products). If other IFU groups include a keyword giving the name of the wavelength table, then it's fine with me to include it (but I doubt that's the case, because it seems no one else uses wavelength tables). It's not needed in order to satisfy any FITS standard requirements.

Reply:
Thanks that makes sense. I got it.